docs/superpowers/specs/2026-04-20-career-ops-saas-ui-design.md

```
@@ -0,0 +1,234 @@
+# Career-Ops SaaS Web App — Design Spec
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)